.github/enabled_plugins

```diff
@@ -0,0 +1 @@
+[rabbitmq_management,rabbitmq_stomp]
```

Comment:
```suggestion
[rabbitmq_stomp]

```

We don't need the management console enabled during tests: I'm almost certain we can't get to view it anyway.